samples/vboxwrapper/vbox_common.cpp

```diff
@@ -661,7 +661,7 @@ int VBOX_BASE::get_scratch_directory(string& dir) {
 int VBOX_BASE::get_slot_directory(string& dir) {
     char slot_dir[256];
 
-    getcwd(slot_dir, sizeof(slot_dir));
+    if (getcwd(slot_dir, sizeof(slot_dir))) {;}
```

Comment:
Maybe instead of this weird looking code is better to check for an error? In case of function failure the content of `slot_dir` might contain thrash and further crash the whole application.

Reply:
Cancelled this change and reverted back to the original code.